src/pages/createOrder.js

```diff
@@ -665,7 +665,7 @@ class CreateOrderPage extends React.Component {
     if (step.name === 'contributeAs') {
       return (
         <Flex justifyContent="center" width={1}>
-          <Box width={[0, null, null, 1 / 4]} />
+          <Box width={[0, null, null, 0.21]} />
```

Comment:
It's still a weird value, what is the rationale behind it?

Reply:
I felt that was the only accurate way, till I remember to read the docs. :slightly_smiling_face: Just changed it to `'24em'`.

Reply:
Screens with big resolution will still have a centering issue (see screenshot below). Also it's unclear to me where does this `24em` comes from.

Why not copy the behaviour that we have for step details, step payment and step summary where we do `width={1/5} | width={3/5} | width={1/5}`?

![Screenshot_2019-05-03 Contribute - Babel - Open Collective](https://user-images.githubusercontent.com/1556356/57138047-ecbda000-6db1-11e9-81b6-e1d4f0e892db.png)

Reply:
@Betree Using `width={1/5}` led to the behavior on Firefox, `24em` appears to be more accurate and less weird value compare to the previous have used.

Reply:
@Betree @flickz I have to merge that feature but I don't think we have an agreement on the `24em` value, we'll need to follow up on that.

Reply:
@flickz `1/5` is easy to read and understand. We could also go with `20/100` if we clearly want to express that it is a percentage.

I still don't understand where does this `24em` comes from.

Opening another issue to track this.